packages/lib/src/Field/useRegisterField.ts

```diff
@@ -12,28 +12,35 @@ export const useRegisterField = (props: FieldCommonProps) => {
   const field = computedField.get();
   const isReady = field !== undefined;
 
-  const formagus = computed<Required<FormagusProps> | undefined>(() => {
+  const formagus = computed<FormagusProps | undefined>(() => {
     if (!isReady) {
       // component is not yet registered
       return undefined;
     }
 
     const {meta, errors} = field;
-    const hasValidation = meta.isMounted && field.props?.onValidate !== undefined;
+
+    const safeErrors = toJSCompat(errors);
+    const safeValue = toJSCompat(field.value, false);
+    const safeCustomState = toJSCompat(meta.customState);
 
     return {
       name: props.name,
-      value: toJSCompat(field.value, false),
+      value: safeValue,
+      errors: safeErrors,
```

Comment:
do we really need same errors on field and meta level

Reply:
nope, but they will be gone from meta level, It's just for the sake of not breaking everything at once. I've added @deprecated comment there. They not gonna be in meta to align placement of errors on field and form level API's. That's what we talked about before, actually